
geometry_msgs/Point location
MoveOptions options
float32 yaw
